.pipelines/templates/create-cluster-swiftv2.yaml

```diff
@@ -0,0 +1,46 @@
+parameters:
+  name: ""
+  displayName: ""
+  clusterType: ""
+  clusterName: "" # Recommended to pass in unique identifier
+  vmSize: ""
+  vmSizeWin: ""
+  k8sVersion: ""
+  osSkuWin: "Windows2022" # Currently we only support Windows2022
+  dependsOn: ""
+  region: ""
+  os: linux
+
+jobs:
+  - job: ${{ parameters.name }}
+    displayName: Cluster - ${{ parameters.name }}
+    steps:
+      - task: AzureCLI@1
+        inputs:
+          azureSubscription: $(ACN_TEST_SERVICE_CONNECTION)
+          scriptLocation: "inlineScript"
+          scriptType: "bash"
+          addSpnToEnvironment: true
+          inlineScript: |
+            set -e
+            echo "Check az version"
+            az version
+            if ${{ lower(contains(parameters.clusterType, 'dualstack')) }}
+            then
+              echo "Install az cli extension preview"
+              az extension add --name aks-preview
+              az extension update --name aks-preview
+            fi
+            mkdir -p ~/.kube/
+            make -C ./hack/aks azcfg AZCLI=az REGION=${{ parameters.region }}
+
+            make -C ./hack/aks ${{ parameters.clusterType }} \
+            AZCLI=az REGION=${{ parameters.region }} SUB=$(SUB_AZURE_NETWORK_AGENT_TEST) \
+            CLUSTER=${{ parameters.clusterName }} \
+            VM_SIZE=${{ parameters.vmSize }} VM_SIZE_WIN=${{ parameters.vmSizeWin }} \
+            OS_SKU_WIN=${{ parameters.osSkuWin }} OS=${{parameters.os}} \
```

Comment:
we're not creating windows nodes yeah?

Reply:
No. Actually this part has never been executed. Let me remove it.